db/db_compaction_test.cc

```diff
@@ -400,6 +400,7 @@ TEST_F(DBCompactionTest, SkipStatsUpdateTest) {
   // The test will need to be updated if the internal behavior changes.
 
   Options options = DeletionTriggerOptions(CurrentOptions());
+  options.disable_auto_compactions = true;
```

Comment:
Is this necessary?

Reply:
Yes; the flakiness is in part caused by a compaction starting right after the open and creating a new `Version` (which results in those stats getting updated).